packages/date-picker/test/validation.test.js

```diff
@@ -13,6 +13,19 @@ class DatePicker2016 extends DatePicker {
 
 customElements.define('vaadin-date-picker-2016', DatePicker2016);
 
+function waitForValueChange(datePicker, callback) {
```

Comment:
Added this function to avoid using `done` callback and instead use `async` / `await` consistently.